src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/WhenToApplyChange.cshtml

```diff
@@ -21,102 +21,35 @@
-                                </div>
-                                <div class="form-group form-group-month">
-                                    <label for="DateOfChange.Month">
-                                        Month
-                                    </label>
-                                    @Html.TextBox("DateOfChange.Month", Model.Data.ChangeStatusViewModel.DateOfChange.Month, new { @class = "form-control length-limit", type = "number", maxlength = "2", min = "1", max = "12", aria_labelledby = "DateOfChange.Month" })
-
-                                </div>
-                                <div class="form-group form-group-year">
-                                    <label for="DateOfChange.Year">
-                                        Year
-                                    </label>
-                                    @Html.TextBox("DateOfChange.Year", Model.Data.ChangeStatusViewModel.DateOfChange.Year, new { @class = "form-control length-limit", type = "number", maxlength = "4", min = "1900", max = "9999", aria_labelledby = "DateOfChange.Year" })
-                                </div>
-                            </div>
-
-                            @if (Model.Data.EarliestDateIsStartDate)
-                            {
-                                <div class="panel panel-border-wide">
-                                    <p class="strong">
-                                        If you want to stop and restart this apprenticeship, enter the original start <br />date: @Model.Data.EarliestDate.ToFullDateEntryFormat().
-                                    </p>
-                                </div>
-                            }
+                        <div class="form-group form-group-year">
+                            <label for="DateOfChange.Year">
+                                Year
+                            </label>
+                            @Html.TextBox("DateOfChange.Year", Model.Data.ChangeStatusViewModel.DateOfChange.Year, new { @class = "form-control length-limit", type = "number", maxlength = "4", min = "1900", max = "9999", aria_labelledby = "DateOfChange.Year" })
                         </div>
+                    </div>
+                </div>
 
-                        <div class="form-group">
-                            <button class="button" type="submit" id="submit-apply-change">Continue</button>
-                            <a class="text-link cancel-link" href=@Url.RouteUrl("OnProgrammeApprenticeshipDetails") aria-label="Cancel">Cancel</a>
-                        </div>
-                    </fieldset>
+                <div class="form-group">
+                    <button class="button" type="submit" id="submit-apply-change">Save and Continue</button>
+                </div>
+            </fieldset>
```

Comment:
The design shows a details component titled 'Error states' below the call to action button (which is not mentioned in the summary of changes). We should ask the designer about it.

Reply:
The error states is not part of the design.  It's what to display on those errors.

Reply:
The 'Error states' component is part of the prototype to switch between the different error states, and shouldn't have been included on the design.